File: class06/cpp/AP0602-MediaAprovacao.cpp
#include <iostream>
#include <iomanip>
int main() {
    double n1, n2;
    std::cout << "Nota 1: ";
    std::cin >> n1;
    std::cout << "Nota 2: ";
    std::cin >> n2;
    double media = (n1 + n2) / 2;
    std::cout << std::fixed << std::setprecision(1) << "Media: " << media << std::endl;
    if (media >= 7) std::cout << "APROVADO" << std::endl;
    else if (media >= 5) std::cout << "RECUPERACAO" << std::endl;
    else std::cout << "REPROVADO" << std::endl;
    return 0;
}

File: class06/java/AP0602-MediaAprovacao.java
import java.util.Scanner;

public class MediaAprovacao {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Nota 1: ");
        double n1 = sc.nextDouble();
        System.out.print("Nota 2: ");
        double n2 = sc.nextDouble();
        double media = (n1 + n2) / 2;
        System.out.printf("Media: %.1f%n", media);
        if (media >= 7) System.out.println("APROVADO");
        else if (media >= 5) System.out.println("RECUPERACAO");
        else System.out.println("REPROVADO");
    }
}
